### ISSUE TITLE
Log: Error while parsing activity event, Image not found

### ISSUE THREAD

**Issue by hopfenhuhn** [nextcloud/circles#1703]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

### Server configuration

**Web server:**

**Database:** MySQL

**PHP version:** 8.2

**Nextcloud version:** 27.1.11

<details>
<summary>List of activated apps</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

<details>
<summary>Nextcloud configuration</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Browser

**Browser name:** Firefox

**Browser version:** 115.15.0esr

**Operating system:** Windows

<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

`{"reqId":"Zvz1RKPfwX3rawV7aSkMOQAAERU","level":4,"time":"2024-10-02T07:24:52+00:00","remoteAddr":"XXX","user":"XXX","app":"activity","method":"GET","url":"/ocs/v2.php/apps/dashboard/api/v2/widget-items?widgets%5B%5D=activity","message":"Error while parsing activity event","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Ge-cko/20100101 Fire-fox/130.0","version":"27.1.11.3","data":{"exception":"{\"class\":\"RuntimeException\",\"message\":\"image not found: image:png webroot: server-root:/www/htdocs/XXX/cloud\",\"code\":0,\"file\":\"/www/htdocs/XXX/cloud/lib/private/URLGenerator.php:274\",\"trace\":\"#0 /www/htdocs/XXX/cloud/apps/circles/lib/Activity/Provider.php(96): OC\\URLGenerator->imagePath('circles', 'png')\\n#1 /www/htdocs/XXX/cloud/apps/circles/lib/Activity/Provider.php(67): OCA\\Circles\\Activity\\Provider->setIcon(Object(OC\\Activity\\Event))\\n#2 /www/htdocs/XXX/cloud/apps/activity/lib/GroupHelper.php(89): OCA\\Circles\\Activity\\Provider->parse('de_DE', Object(OC\\Activity\\Event), Object(OC\\Activity\\Event))\\n#3 /www/htdocs/XXX/cloud/apps/activity/lib/GroupHelper.php(76): OCA\\Activity\\GroupHelper->addEvent(5350, Object(OC\\Activity\\Event))\\n#4 /www/htdocs/XXX/cloud/apps/activity/lib/Data.php(260): OCA\\Activity\\GroupHelper->addActivity(Array)\\n#5 /www/htdocs/XXX/cloud/apps/activity/lib/Dashboard/ActivityWidget.php(152): OCA\\Activity\\Data->get(Object(OCA\\Activity\\GroupHelper), Ob-ject(OCA\\Activity\\UserSettings), 'XXX', 0, 49, 'desc', 'by', '', 0)\\n#6 /www/htdocs/XXX/cloud/apps/dashboard/lib/Controller/DashboardApiController.php(134): OCA\\Activity\\Dashboard\\ActivityWidget->getItemsV2(XXX', NULL, 7)\\n#7 /www/htdocs/XXX/cloud/lib/private/AppFramework/Http/Dispatcher.php(230): OCA\\Dashboard\\Controller\\DashboardApiController->getWidgetItemsV2(Array, 7, Array)\\n#8 /www/htdocs/XXX/cloud/lib/private/AppFramework/Http/Dispatcher.php(137): OC\\AppFramework\\Http\\Dispatcher->executeController(Object(OCA\\Dashboard\\Controller\\DashboardApiController), 'getWidgetI-temsV...')\\n#9 /www/htdocs/XXX/cloud/lib/private/AppFramework/App.php(183): OC\\AppFramework\\Http\\Dispatcher->dispatch(Object(OCA\\Dashboard\\Controller\\DashboardApiController), 'getWidgetI-temsV...')\\n#10 /www/htdocs/XXX/cloud/lib/private/Route/Router.php(315): OC\\AppFramework\\App::main('OCA\\\\Dashboard\\\\C...', 'getWidgetItemsV...', Ob-ject(OC\\AppFramework\\DependencyInjection\\DIContainer), Array)\\n#11 /www/htdocs/XXX/cloud/ocs/v1.php(65): OC\\Route\\Router->match('/ocsapp/apps/da...')\\n#12 /www/htdocs/XXX/cloud/ocs/v2.php(23): require_once('/www/htdocs/w01...')\\n#13 {main}\"}","app":"activity"},"id":"66fcf5451ef0b"}`

**Comment by hopfenhuhn** [comment]
`{"reqId":"Zvz1RKPfwX3rawV7aSkMOQAAERU","level":4,"time":"2024-10-02T07:24:52+00:00","remoteAddr":"XXX","user":"XXX","app":"activity","method":"GET","url":"/ocs/v2.php/apps/dashboard/api/v2/widget-items?widgets%5B%5D=activity","message":"Error while parsing activity event","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Ge-cko/20100101 Fire-fox/130.0","version":"27.1.11.3","data":{"exception":"{\"class\":\"RuntimeException\",\"message\":\"image not found: image:png webroot: server-root:/www/htdocs/XXX/cloud\",\"code\":0,\"file\":\"/www/htdocs/XXX/cloud/lib/private/URLGenerator.php:274\",\"trace\":\"#0 /www/htdocs/XXX/cloud/apps/circles/lib/Activity/Provider.php(96): OC\\URLGenerator->imagePath('circles', 'png')\\n#1 /www/htdocs/XXX/cloud/apps/circles/lib/Activity/Provider.php(67): OCA\\Circles\\Activity\\Provider->setIcon(Object(OC\\Activity\\Event))\\n#2 /www/htdocs/XXX/cloud/apps/activity/lib/GroupHelper.php(89): OCA\\Circles\\Activity\\Provider->parse('de_DE', Object(OC\\Activity\\Event), Object(OC\\Activity\\Event))\\n#3 /www/htdocs/XXX/cloud/apps/activity/lib/GroupHelper.php(76): OCA\\Activity\\GroupHelper->addEvent(5350, Object(OC\\Activity\\Event))\\n#4 /www/htdocs/XXX/cloud/apps/activity/lib/Data.php(260): OCA\\Activity\\GroupHelper->addActivity(Array)\\n#5 /www/htdocs/XXX/cloud/apps/activity/lib/Dashboard/ActivityWidget.php(152): OCA\\Activity\\Data->get(Object(OCA\\Activity\\GroupHelper), Ob-ject(OCA\\Activity\\UserSettings), 'XXX', 0, 49, 'desc', 'by', '', 0)\\n#6 /www/htdocs/XXX/cloud/apps/dashboard/lib/Controller/DashboardApiController.php(134): OCA\\Activity\\Dashboard\\ActivityWidget->getItemsV2(XXX', NULL, 7)\\n#7 /www/htdocs/XXX/cloud/lib/private/AppFramework/Http/Dispatcher.php(230): OCA\\Dashboard\\Controller\\DashboardApiController->getWidgetItemsV2(Array, 7, Array)\\n#8 /www/htdocs/XXX/cloud/lib/private/AppFramework/Http/Dispatcher.php(137): OC\\AppFramework\\Http\\Dispatcher->executeController(Object(OCA\\Dashboard\\Controller\\DashboardApiController), 'getWidgetI-temsV...')\\n#9 /www/htdocs/XXX/cloud/lib/private/AppFramework/App.php(183): OC\\AppFramework\\Http\\Dispatcher->dispatch(Object(OCA\\Dashboard\\Controller\\DashboardApiController), 'getWidgetI-temsV...')\\n#10 /www/htdocs/XXX/cloud/lib/private/Route/Router.php(315): OC\\AppFramework\\App::main('OCA\\\\Dashboard\\\\C...', 'getWidgetItemsV...', Ob-ject(OC\\AppFramework\\DependencyInjection\\DIContainer), Array)\\n#11 /www/htdocs/XXX/cloud/ocs/v1.php(65): OC\\Route\\Router->match('/ocsapp/apps/da...')\\n#12 /www/htdocs/XXX/cloud/ocs/v2.php(23): require_once('/www/htdocs/w01...')\\n#13 {main}\"}","app":"activity"},"id":"66fcf5451ef0b"}`

**Comment by joshtrichards** [comment]
Fixed in #1544 / #1620

> Nextcloud version: 27.1.11

You're running an end-of-life Nextcloud release (27):

https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule
https://docs.nextcloud.com/server/stable/admin_manual/release_schedule.html
